### PR TITLE
Blog post: scribble/html demo

### PR DESCRIPTION
**Pull request by bennn**
- demo the `scribble/html` language
- explain a little how the PRL website works 
- fix some typos in the website

### CODE REVIEW

File: blog/_src/posts/2017-05-19-scribble-html.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+ "field" of the person:
+
+```
+@person[#:name "Ben Greenman"
+        #:title "Advisor: Matthias Felleisen"
+        #:e-mail "types@ccs.neu.edu"
+        #:website "http://ccs.neu.edu/home/types"
+        #:history @list["Joined 2014"]
+        #:img "ben_greenman.jpg"]{
+  I like constructions ...
```
Comment (gasche) [inline blog/_src/posts/2017-05-19-scribble-html.md]: Pure nitpick, but it's a bit surprising that you show content here that is not in the version above. Maybe include "I like constructions..." on both sides?

**Comment by gasche** [comment]
I found it excellent, thanks!

File: blog/_src/posts/2017-05-19-scribble-html.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+@require[(only-in racket/list add-between)]
+
+@(define (vertical-text . str*)
+   (add-between (string->list (append* str*)) (br)))
+```
+
+
+#### 2. More Structure, Less Boilerplate
+
+Here's part of the old definition of "Ben Greenman" on the [People page](file:///Users/ben/code/racket/website/people.html):
```
Comment (mhyee) [inline blog/_src/posts/2017-05-19-scribble-html.md]: Link should be `http://prl.ccs.neu.edu/people.html`.

**Comment by bennn** [comment]
thanks for the comments

**Pull request merged**